webv2/web.go

```diff
@@ -734,6 +734,122 @@ type DropDetail struct {
 	Name string `json:"Name"`
 }
 
+func restoreTable(w http.ResponseWriter, r *http.Request) {
```

Comment:
New note appears for the table regarding foreign key name : https://drive.google.com/file/d/1Yhpq78LpDsz1zzTwP3e4NQhnoqPa-DLb/view?usp=sharing&resourcekey=0-TyTrw0L9mMWVuhxMrwwhGg

Steps to reproduce:
1. Delete table 1 then table2
2. Restore table 2 then table 1

Reply:
Fixed